geode-assembly/src/test/java/org/apache/geode/management/internal/cli/commands/StatusLocatorRealGfshTest.java

```diff
@@ -30,7 +30,6 @@
   @Test
```

Comment:
Even GitHub knows I'm being ridiculous and won't let me tag up in line 23, where you didn't do anything...

But as long as we're in here, imports should be alphabetized.  Line 23's `import org.apache.geode.test.junit.categories.AcceptanceTest` should be at the top of the `org.apache` block.

Also, `geode-core/src/main/java/org/apache/geode/management/internal/cli/remote/OnlineCommandProcessor.java` and `geode-core/src/test/java/org/apache/geode/management/internal/cli/remote/OnlineCommandProcessorTest.java` have unused imports.